### PR TITLE
Added Redgate/Flyway

### PR DESCRIPTION
**Pull request by Roemer**
Thank you for your contribution! Some pointers to get it merged quickly:

For contributions in `software/`:

  - Status: please select a value from the status table at the top
  - Version: Status Vulnerable / Workaround? -> List vulnerable versions.
             Status Fix?                     -> List fixed versions.
  - Links: make sure you link to a public statement by the developer.
    Alternatively, include and link a file in the `software/vendor-statements/`
    directory
  - Please mind the sorting
  - Please put vendor/product name in PR title (instead of "Update README.md")

### CODE REVIEW

File: software/README.md [hunk 1]
```diff
@@ -1587,6 +1587,7 @@ The `Version` relates to the `Status` column. If `Status` is Vulnerable, Version
 | Red Hat         | Red Hat OpenStack Platform 13 (Queens) opendaylight |  | Vulnerable | | [source](https://access.redhat.com/security/cve/cve-2021-44228) |
 | Red Hat         | Red Hat Process Automation 7 |  | Vulnerable | | [source](https://access.redhat.com/security/cve/cve-2021-44228) |
 | Red Hat         | Red Hat Single Sign-On 7 |  | Not Vuln | | [source](https://access.redhat.com/security/cve/cve-2021-44228) |
+| Redgate | Flyway |  | Vulnerable | Only vulnerable when using non-default config. | [source](https://flywaydb.org/blog/flyway-log4j-vulnerability) |
```
Comment (maertsen) [inline software/README.md:1590]: Can you add version numbers?

**Review by maertsen — CHANGES_REQUESTED** [review]
hi @Roemer, thanks for your contribution! Can you have a look at my comment?

**Comment by Roemer** [comment]
> Added the version number. As it is up to a user of flyway to use any library he wants, it is potentially an issue for all versions.

**Comment by maertsen** [comment]
I have not spent very much time reading the source, but it appeared to me that their latest release refuses to link against old versions? Should that be reflected in your contribution?

**Comment by Roemer** [comment]
Flyway 8.2.2 forces log4j >= 2.15 so the issue is not fixed yet right? Only as soon as it forces >= 2.16 it is fixed I assume?

**Pull request merged**